src/index.js

```diff
@@ -1 +1,4 @@
 import './scss/index.scss';
+import mainUi from './modules/MainUi.js';
+
+window.addEventListener('DOMContentLoaded', mainUi.setup);
```

Comment:
[Optional] Wow, it works, but what do you think about the `windows.onload = mainUi.setup` will be more straightforward instead of adding an event listener?

Reply:
@aimalamiri 
After reading this [article](https://javascript.info/onload-ondomcontentloaded) I decided to use `DOMContentLoaded` instead of the `load` event. There's not ondomcontentload.
On the other hand. On a previous project I faced the problem with one specific event that didn't work with the .onevent, but it works well with the addEventListener. Sadly to say, I can't recall what event it was. But I decided to go the safe path of using addEventListener, to avoid facing this kind of problem again.